src/Entities/Misc/DreamStrawberry.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Celeste.Mod.Entities;
+using Microsoft.Xna.Framework;
+using Monocle;
+using MonoMod.Utils;
+using System;
+using System.Reflection;
+
+namespace Celeste.Mod.CommunalHelper.Entities {
+
+    // Originally I made this as a standalone entity for someone's map they were working on, but to make this fully work with DreamTunnelDash I moved it to CommunalHelper
+    // I gave them a plugin for the old version when i finished and I'd like to keep some compatability to the old version so they dont have to redo their berries using it
+    [CustomEntity("CommunalHelper/DreamStrawberry", "DreamDashListener/DreamDashBerry")]
+    public class DreamStrawberry : Strawberry {
+
+        /*
+         * Hey! snowii here
+         * 
+         * If anyone is reading this and wants to touch my unholy code AND has a sprite for this, please add it im begging
+         * 
+         * Hope someone finds this super niche excuse for an entity useful
+         */
+
+        private MethodInfo OriginalOnDash;
+
+        public static Color[] DreamTrailColors = new Color[] {
+            Calc.HexToColor("FFEF11"),
+            Calc.HexToColor("08A310"),
+            Calc.HexToColor("FF00D0"),
+            Calc.HexToColor("5FCDE4"),
+            Calc.HexToColor("E0564C")
+        };
+
+        public static DynamicData dreamStrawberrySpriteData;
```

Comment:
You probably don't want this to be `static`. Each time a Dream Stawberry is instantiated, you are setting this field to a value which takes the instance of that entity. So basically, everytime you access this field, it's the data of the latest instantiated Dream Strawberry, and not the one you actually want.
(Also the name is a bit misleading, I'd call it `strawberryData` instead)

Reply:
oh yeah that's a good point, will fix these too